.github/workflows/docs-on-pr.yml

```diff
@@ -23,16 +23,14 @@ jobs:
          uses: actions/cache@v2
          with:
            path: .venv
-           key: venv-${{ runner.os }}-${{ hashFiles('**/poetry.lock') }}
-
+           key: venv-${{ runner.os }}-docs-${{ hashFiles('**/poetry.lock') }}
        - name: Install dependencies
          if: steps.cached-poetry-dependencies.outputs.cache-hit != 'true'
          run: |
-           poetry install -E all -vv
-       - name: Install docs dependencies
+           poetry install -E "all docs" -vv
```

Comment:
Why do we need to install "all" dependencies just to build the docs?

Reply:
New behavior - poetry remove all dependencies from previous steps. So you need to specify all extras at once